app/models/contract.rb

```diff
@@ -0,0 +1,18 @@
+# A contract is a ...
+# recruitment_target:
+# output_payment_percentage:
+class Contract < ApplicationRecord
+  belongs_to :statement
+  belongs_to :course
+  belongs_to :cohort
+  belongs_to :lead_provider
+
+  validates :per_participant, numericality: { greater_than: 0 }
+  validates :recruitment_target, numericality: { only_integer: true, greater_than: 0 }
+  validates :output_payment_percentage, numericality: { only_integer: true, greater_than: 0, less_than: 100 }
+
+  # these three fields should be null after [a certain date]
+  validates :number_of_payment_periods, numericality: { only_integer: true, greater_than: 0 }
+  validates :service_fee_percentage, numericality: { only_integer: true, greater_than: 0 }
+  validates :service_fee_installments, numericality: { only_integer: true, greater_than: 0 }
```

Comment:
Maybe some/all of these should allow zero.

Reply:
Good call, in ECF most are [greater than or equal to](https://github.com/DFE-Digital/early-careers-framework/blob/main/app/models/npq_contract.rb#L8), but `recruitment_target` and `per_participant` are greater than.

Reply:
discussed in person: let's mirror what's already in ECF